components/tag-list/tag-list-item-mixin.js

```diff
@@ -200,13 +203,16 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 		));
 	}
 
-	_renderTag(tagContent, hasTruncationTooltip) {
+	_renderTag(tagContent, hasTruncationTooltip, description) {
 		const buttonText = typeof tagContent === 'object'
 			? this.localize('components.tag-list.clear', { value: '' })
 			: this.localize('components.tag-list.clear', { value: tagContent });
+		const hasDescription = !!description;
+		const tooltipHeader = hasDescription ? html`<h4>${tagContent}</h4>` : tagContent;
```

Comment:
Note: with Chrome + VO, the tooltip content is not announced when the tooltip contains the `h4` element. I think @dcostiuc ran into similar behaviour with `d2l-tooltip-help` when lists were placed inside. Perhaps we just want to use one of our heading styles, and forego the semantic heading. From what I gather, this might be due to VO strictly adhering to the idea that [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) element references should only contain text (no complex elements).

Reply:
Sounds good, I'll swap it out for a span with the heading styles 👍🏻